<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KUserFeedback::Console::AnalyticsView</class>
 <widget class="QWidget" name="KUserFeedback::Console::AnalyticsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>353</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="chartTab">
      <attribute name="title">
       <string>Charts</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&amp;Aggregation:</string>
           </property>
           <property name="buddy">
            <cstring>chartType</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="chartType"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="chartStack">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="timelinePage">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QChartView" name="timelineChartView">
             <property name="renderHints">
              <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="singularPage">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QChartView" name="singularChartView"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QSlider" name="timeSlider">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="timeLabel"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="aggregationTab">
      <attribute name="title">
       <string>Aggregated Data</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QTableView" name="aggregatedDataView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="dataTab">
      <attribute name="title">
       <string>Raw Data</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTableView" name="dataView">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionAggregateYear">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Year</string>
   </property>
   <property name="toolTip">
    <string>Aggregate data by year.</string>
   </property>
  </action>
  <action name="actionAggregateMonth">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Month</string>
   </property>
   <property name="toolTip">
    <string>Aggregate data by month.</string>
   </property>
  </action>
  <action name="actionAggregateWeek">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Week</string>
   </property>
   <property name="toolTip">
    <string>Aggregate data by week.</string>
   </property>
  </action>
  <action name="actionAggregateDay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Day</string>
   </property>
   <property name="toolTip">
    <string>Aggregate data by day.</string>
   </property>
  </action>
  <action name="actionExportData">
   <property name="icon">
    <iconset theme="document-export"/>
   </property>
   <property name="text">
    <string>&amp;Export Data...</string>
   </property>
   <property name="toolTip">
    <string>Export all recorded data.</string>
   </property>
  </action>
  <action name="actionImportData">
   <property name="icon">
    <iconset theme="document-import"/>
   </property>
   <property name="text">
    <string>&amp;Import Data...</string>
   </property>
   <property name="toolTip">
    <string>Import data samples.</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset theme="view-refresh"/>
   </property>
   <property name="text">
    <string>&amp;Reload Data</string>
   </property>
   <property name="toolTip">
    <string>Refresh data from the server.</string>
   </property>
  </action>
  <action name="actionTimelineChart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="office-chart-line-stacked"/>
   </property>
   <property name="text">
    <string>&amp;Timeline Chart</string>
   </property>
   <property name="toolTip">
    <string>Show development over time.</string>
   </property>
  </action>
  <action name="actionSingularChart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="office-chart-pie"/>
   </property>
   <property name="text">
    <string>&amp;Singular Chart</string>
   </property>
   <property name="toolTip">
    <string>Show chart for a single time interval.</string>
   </property>
  </action>
  <action name="actionExportChart">
   <property name="icon">
    <iconset theme="document-save-as"/>
   </property>
   <property name="text">
    <string>Export &amp;Chart...</string>
   </property>
   <property name="toolTip">
    <string>Export the current chart as image, SVG or PDF.</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QChartView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
